netdeploy/remote/nodecfg/nodeConfigurator.go

```diff
@@ -134,6 +149,23 @@ func (nc *nodeConfigurator) prepareNodeDirs(configs []remote.NodeConfig, rootCon
 			return
 		}
 
+		// Copy the bootstrapped files into current ledger folder
+		if nc.bootstrappedBlockFile != "" {
+			fmt.Fprintf(os.Stdout, "... copying block database file to ledger folder ...\n")
+			_, err = util.CopyFile(nc.bootstrappedBlockFile, filepath.Join(nodeDest, genesisDir, fmt.Sprintf("%s.block.sqlite", config.LedgerFilenamePrefix)))
+			if err != nil {
+				return
+			}
+		}
+
+		if nc.bootstrappedTrackerFile != "" {
```

Comment:
the `nc.bootstrappedBlockFile` and `nc.bootstrappedTrackerFile` should be deployed together, or not at all.
so, if you have only one of them, either fail right away, or issue a warning message and don't copy just one file.